web-ui/src/index.js

```diff
@@ -3,10 +3,12 @@ import ReactDOM from "react-dom";
 import "./index.css";
 import App from "./App";
 import * as serviceWorker from "./serviceWorker";
+import { BrowserRouter as Router } from "react-router-dom";
 
 ReactDOM.render(
-    <App />, 
-  document.getElementById("root")
+  <Router>
```

Comment:
We don't need this at the root because it's already used in App.js. Delete this addition of Router.